src/env_variables.yaml

```diff
@@ -1,3 +1,4 @@
 # This is for dev, and will get overwritten when deploying to prod
 env_variables:
   FLASK_SECRET_KEY: "thebluealliance"
+  CD_REQUEST_USER_AGENT: "default"
```

Comment:
I think you need to update `ops/deploy/update_env_variables_yaml.sh` and `.github/workflows/push.yml`